pcsx2-qt/QtHost.cpp

```diff
@@ -240,31 +205,45 @@ void EmuThread::startVM(std::shared_ptr<VMBootParameters> boot_params)
 		return;
 	}
 
-	pxAssertRel(!VMManager::HasValidVM(), "VM is shut down");
-
 	// Determine whether to start fullscreen or not.
 	m_is_rendering_to_main = shouldRenderToMain();
 	if (boot_params->fullscreen.has_value())
 		m_is_fullscreen = boot_params->fullscreen.value();
 	else
 		m_is_fullscreen = Host::GetBaseBoolSettingValue("UI", "StartFullscreen", false);
 
-	if (!VMManager::Initialize(*boot_params))
-		return;
+	const auto hardcore_disable_callback = [](std::string reason, VMBootRestartCallback restart_callback) {
```

Comment:
Noticed an issue here: These callbacks are const even though I try to move them later, so that'll do an unintended copy.

Reply:
Fixed. Also noticed that the rebase I did earlier ate a commit, that's fixed too.